### DISCUSSION TITLE
INSTALLATION OF OKD ON PHYSICAL SERVER

### DISCUSSION THREAD

**Discussion by Pwdak** [okd-project/okd#1756] [Q&A] [answered]
Hi! Hope all are doing well! 
Can I get with you setps to install okd on physical server ? Thanks

**Comment by rvanderp3** [comment, marked as answer]
I would recommend starting here: https://docs.okd.io/latest/installing/installing_bare_metal/preparing-to-install-on-bare-metal.html
